k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -137,9 +137,7 @@ function deleteCortxServer()
     printf "########################################################\n"
     printf "# Delete CORTX Server                                  #\n"
     printf "########################################################\n"
-    for node in "${node_name_list[@]}"; do
-        uninstallHelmChart "cortx-server-${node}-${namespace}" "${namespace}"
-    done
+    uninstallHelmChart "cortx-server-${namespace}" "${namespace}"
```

Comment:
For now, better to do both uninstalls so previous deployments can be uninstalled w/o concern of which version is being used.